Python/Tests/Core/WorkspaceInterpreterFactoryTests.cs

```diff
@@ -211,9 +211,9 @@ public class WorkspaceInterpreterFactoryTests {
             return new WorkspaceTestHelper.MockWorkspaceContext(new WorkspaceTestHelper.MockWorkspace(workspacePath));
         }
 
-        private static void CreatePythonVirtualEnv(string pythonInterpreterPath, string workspacePath, string envName) {
+        internal static void CreatePythonVirtualEnv(string pythonInterpreterPath, string workspacePath, string envName, string environmentCreationParameter = "venv") {
```

Comment:
`virtualEnvModule` would be a more descriptive name for what it is you are passing in, rather than `environmentCreationParameter`